src/components/OSCALLoader.js

```diff
@@ -56,8 +56,10 @@ export default function OSCALLoader(props) {
   };
 
   const handleReloadClick = () => {
-    setIsLoaded(false);
-    loadOscalData(oscalUrl);
+    if (isLoaded) {
```

Comment:
Should we consider also visually disabling the reload button as part of this?

Reply:
Good idea, I'll look into that.

Reply:
Should be able to just set the `disabled` flag: https://material-ui.com/components/buttons/